conftest.py

```diff
@@ -3,6 +3,7 @@
 from page_objects.PageFactory import PageFactory
 from conf import browser_os_name_conf
 from conf import base_url_conf
+from conf import report_portol_conf
```

Comment:
@Annapooraniqxf2  - you have a typo. Its `portal` not `portol`. Can you please fix? You can use `git mv` to rename the file.

Reply:
Fixed it